mars_rover/spec/traveller_spec.rb

```diff
@@ -4,23 +4,21 @@
     context "Given a traveller instance" do
       it "should set their location and path" do
         location = Location.new(1,2,'N')
-        traveller = Traveller.new(location, "LMLMM")
+        direction = Direction.new("LMLMM")
+        traveller = Traveller.new(location, direction)
+
         expect(location.x).to eq (1)
         expect(location.y).to eq (2)
         expect(location.compass_point).to eq ('N')
-        expect(traveller.path).to eq (["L","M","L","M","M"])
+        expect(direction.moves).to eq (["L","M","L","M","M"])
```

Comment:
`(...)` interpreted as grouped expression.<br>Space missing after comma.<br>Use `%w` or `%W` for array of words.